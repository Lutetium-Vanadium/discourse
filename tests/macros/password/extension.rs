fn main() {
    let q = discourse::questions![Password { extension: todo!() }];
}
