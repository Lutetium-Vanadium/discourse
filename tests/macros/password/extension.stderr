error: option `extension` does not exist for kind `password`
 --> $DIR/extension.rs:2:46
  |
2 |     let q = discourse::questions![Password { extension: todo!() }];
  |                                              ^^^^^^^^^
